system/Pager/PagerRenderer.php

```diff
@@ -76,6 +76,13 @@ class PagerRenderer
 	 */
 	protected $pageSelector;
 
+	/**
+	 * Links to dipsplay
+	 *
+	 * @var array
+	 */
+	private $links = [];
```

Comment:
`$last` technically can be `< 1` so one empty array is a valid return for `links()` already. This should start as `null` and be checked as such for processing.
```suggestion
	 * @var array|null
	 */
	private $links;
```